Gemfile

```diff
@@ -33,6 +33,7 @@ group :development, :test do
   gem "byebug"
   gem "factory_girl_rails"
   gem "faker"
+  gem "ffaker"
```

Comment:
`ffaker` functionality is not a superset of `faker` functionality. I don't think `FFaker::App`exists whereas we are using  `Faker::App`.